README.md

```diff
@@ -107,10 +107,10 @@ If you get a 500 local server error, run:
 ```
 yarn start --reset-cache
 ``` 
-**Note:**
-To use the EN APIs on Android the Google account on your Android device must be approved. Reach out to the PatchCheck contacts at Google to get added to the list.
-
-Device storage can be cleared by long-pressing on the app icon in the simulator, clicking "App info", then "Storage", and lastly, "Clear Storage".
+**Testing:**
+- To use the EN APIs on Android the Google account on your Android device must be approved. Reach out to the PatchCheck contacts at Google to get added to the list.
+- To provide dianogisis keys to the API the applicationId should be whitelisted, you can skip that verification step by checking `Settings > Google > COVID-19 Exposure Notifications >  Debug mode > Bypass app signature check`
+- Device storage can be cleared by long-pressing on the app icon in the simulator, clicking "App info", then "Storage", and lastly, "Clear Storage".
```

Comment:
Can you please provide more explanation on why a someone would want to clear storage?

Reply:
I haven't added that line, it was already in the documentation
I think it could be useful to wipe the database and make it display the notification again